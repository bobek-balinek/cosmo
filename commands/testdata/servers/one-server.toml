[servers.server-1]
host = 'host'
user = 'user'

  [servers.server-1.tasks.task-1]
    remote = [
      { exec = 'echo task-1' },
    ]

  [servers.server-1.tasks.task-2]
    remote = [
      { exec = 'echo task-2' },
    ]
