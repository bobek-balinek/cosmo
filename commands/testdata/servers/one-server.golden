server-1 user@host tasks: 2
